flask/auth.py

```diff
@@ -129,10 +129,18 @@ def logout():
 @swag_from("documentation/reset_password.yaml", methods=["POST"])
 def reset_password():
     password = request.form.get("password")
+    oldpassword = request.form.get("oldpassword")
     uuid = request.form.get("uuid")
```

Comment:
Can you check if uuid is not None ?

Reply:
we have the mongodb to handle , if uuid is none then the query would be none.